cmd/clusteradm/clusteradm.go

```diff
@@ -75,29 +82,39 @@ func main() {
 		{
 			Message: "General commands:",
 			Commands: []*cobra.Command{
+				create.NewCmd(clusteradmFlags, streams),
+				deletecmd.NewCmd(clusteradmFlags, streams),
+				get.NewCmd(clusteradmFlags, streams),
+				install.NewCmd(clusteradmFlags, streams),
+				upgrade.NewCmd(clusteradmFlags, streams),
 				version.NewCmd(clusteradmFlags, streams),
 			},
 		},
 		{
 			Message: "Registration commands:",
 			Commands: []*cobra.Command{
-				get.NewCmd(clusteradmFlags, streams),
-				deletecmd.NewCmd(clusteradmFlags, streams),
+				acceptclusters.NewCmd(clusteradmFlags, streams),
 				clean.NewCmd(clusteradmFlags, streams),
 				inithub.NewCmd(clusteradmFlags, streams),
 				joinhub.NewCmd(clusteradmFlags, streams),
 				unjoin.NewCmd(clusteradmFlags, streams),
+			},
+		},
+		{
+			Message: "Customized Resource command:",
```

Comment:
that is a bit weird to say it is Customized Resource command

Reply:
how about `Cluster Management commands`?

Reply:
yeah, that is better